src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/config/JurisdictionToUserMappingTest.java

```diff
@@ -41,4 +41,12 @@ public void should_throw_exception_if_not_found() {
             .isInstanceOf(NoUserConfiguredException.class)
             .hasMessage("No user configured for jurisdiction: none");
     }
+
+    @Test
+    public void should_throw_exception_if_none_configured() {
```

Comment:
should we say `should_throw_no_user_configured_exception_when_jurisdiction_is_not_found` ?

Reply:
Not really this is saying that if there is no configuration it should throw the exception.

Reply:
done